pkg/api/v20200430/openshiftcluster_convert.go

```diff
@@ -51,8 +51,15 @@ func (c openShiftClusterConverter) ToExternal(oc *api.OpenShiftCluster) interfac
 	}
 
 	if oc.Properties.WorkerProfiles != nil {
-		out.Properties.WorkerProfiles = make([]WorkerProfile, 0, len(oc.Properties.WorkerProfiles))
-		for _, p := range oc.Properties.WorkerProfiles {
+		workerProfiles := oc.Properties.WorkerProfiles
+
+		// Use enriched worker profile data when available
+		if oc.Properties.WorkerProfilesStatus != nil {
+			workerProfiles = oc.Properties.WorkerProfilesStatus
+		}
+
+		out.Properties.WorkerProfiles = make([]WorkerProfile, 0, len(workerProfiles))
+		for _, p := range workerProfiles {
```

Comment:
Wouldn't this generate new examples for the older API?

Reply:
Nope, because under _openshiftcluster_example.go_ package we set the _WorkerProfilesStatus_ field to nil before calling the converter function (ToExternal). This will ensure that the swagger examples remain unchanged. 

Older APIs swagger examples never showed the enriched worker profile data under _workerProfiles_ field. Do I decided to keep it same.